landscape/client/broker/tests/helpers.py

```diff
@@ -39,14 +39,16 @@ def set_up(self, test_case):
         log_dir = test_case.makeDir()
         test_case.config_filename = os.path.join(test_case.makeDir(),
                                                  "client.conf")
-        open(test_case.config_filename, "w").write(
-            "[client]\n"
-            "url = http://localhost:91919\n"
-            "computer_title = Some Computer\n"
-            "account_name = some_account\n"
-            "ping_url = http://localhost:91910\n"
-            "data_path = %s\n"
-            "log_dir = %s\n" % (data_path, log_dir))
+
+        with open(test_case.config_filename, "w") as fh:
+            fh.write(
```

Comment:
It looks like I changed that code, but I only closed the file handle, to prevent a bunch of warnings when running test cases. I am just going to leave since I shouldn't change it in this pull request.